src/views/PluginList.tsx

```diff
@@ -269,13 +294,13 @@ class PluginList extends ComponentEx<IProps, IComponentState> {
         title: 'Mark as Light',
         action: this.eslifySelected,
         condition: (instanceIds: string[]) =>
-          supportsESL(this.props.gameMode)
+          this.props.supportsESL(this.props.gameMode)
           && (instanceIds.find(pluginId => {
             const plugin = this.state.pluginsCombined[pluginId];
             return plugin.isValidAsLightPlugin
                 && !plugin.isLight
                 && plugin.deployed
-                && path.extname(pluginId) === '.esp';
+                && this.props.pathExtname(pluginId) === '.esp';
```

Comment:
It would have been fine use endsWith here, that way it's easier to actually test this component because now a test environment has to reimplement path.extname

Reply:
I have updated all instances of `path.extname`